scripts/install2

```diff
@@ -126,6 +129,7 @@ elif [[ "$BOARD_HARDWARE" == 'nanopineo2' || "$BOARD_HARDWARE" == 'sun50iw1p1' |
 else
     BOARD_FAMILY="Raspberry Pi"
     BOARD_REVISION="$(sed -rn 's/Revision\s+\:\s+([0-9a-z_\-\s\,\(\)]+)/\1/p' /proc/cpuinfo)"
+    CHANGE_PASSWORD="pi"
```

Comment:
There is forcing password change when running the script. Perhaps the user already changed the password? Can we detect for default password?

Reply:
Hard coded default password hash and check it against `/etc/shadow`